tests/schemas/api_v3/voter.schema.js

```diff
@@ -26,6 +26,8 @@ const votes = {
 	delegateAddress: Joi.string().pattern(regex.ADDRESS_LISK32).required(),
 	name: Joi.string().pattern(regex.NAME).optional(),
 	amount: Joi.string().required(),
+	rank: Joi.number().integer().min(1).required(),
+	voteWeight: Joi.string().min(10).required(),
```

Comment:
This can be updated to use a regex where `regex.VOTE_WEIGHT = /^\b[1-9]\d*000000000\b$/`.
```suggestion
	voteWeight: Joi.string().pattern(regex.VOTE_WEIGHT).required(),
```